### ISSUE TITLE
Control object attributes behaviour

### ISSUE THREAD

**Issue by IvanElenskiy** [mz-automation/libiec61850#122]
Didn't find a proper way to set stSeld, opRcvd, opOk and tOpOk while executing control process. It can be set with checkHandler, but i can't reset stSeld to false. I think it's worth to set these values within control.c functions (Control_writeAccessControlObject/Control_readAccessControlObject) because these attributes are parts of a control object itself.

**Comment by mzillgith** [comment]
Yes you are right. "stSeld" can only be handled internally and the other attributes are also better solved by the control.c code. I already have a ticket for that.

**Comment by IvanElenskiy** [comment]
@mzillgith I didn't find operTimeout in ControlObject required for enhanced security control model. I can't stop returning CONTROL_RESULT_WAITING value from handler because there's no timeout. By the way i think timeout should be controlled by ControlObject itself with corresponding StVal.

**Comment by mzillgith** [comment]
In the latest v1.4 branch stSeld, opRcvd, opOk, and tOpOk are now handled by the library.

**Comment by virfis** [comment]
How to check origin value (orCat, orIdent), when control command received?

**Comment by mzillgith** [comment]
> How to check origin value (orCat, orIdent), when control command received?

In version 1.4 there are now two new functions _ControlAction_getOrCat_ and _ControlAction_getOrIdent_ that can be used to check these values.

**Comment by virfis** [comment]
controlObject->checkHandler take wrong origin values. Its update later with updateControlParameters().
